alert-platform/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'java-platform'
+}
+
+dependencies {
+    constraints {
+        api 'com.github.springtestdbunit:spring-test-dbunit:1.3.0'
+        api 'org.testcontainers:postgresql:1.12.5'
+        api 'org.apache.activemq:activemq-broker:5.15.10'
+
+        api 'org.jsoup:jsoup:1.9.2'
+        api 'com.jayway.jsonpath:json-path:2.3.0'
+        api 'com.sun.mail:javax.mail:1.6.2'
+        api 'javax.mail:javax.mail-api:1.6.2'
+
+        api 'org.postgresql:postgresql:42.2.8'
+        api 'org.liquibase:liquibase-core:3.8.9'
+
+        api 'com.synopsys.integration:blackduck-common:49.2.0'
+        api 'com.synopsys.integration:integration-rest:5.1.1'
+        api 'com.synopsys.integration:integration-common:23.0.0'
+
+        api 'io.springfox:springfox-swagger2:2.8.0'
+        api 'io.springfox:springfox-swagger-ui:2.8.0'
```

Comment:
Would you be willing to test if this can be a `runtime` dependency rather api? 

1. Start Alert
2. Navigate to the "About Page" (the page you are redirected to by default upon login)
3. Click on the link to API Documentation
4. See if it still works

Reply:
Yup, it works.